### PR TITLE
uplode_t3_t4

### PR DESCRIPTION
**Pull request by asifnaseebi**

### CODE REVIEW

File: task/t3.txt [hunk 1]
```diff
@@ -0,0 +1,8 @@
+Sir I searched vim editor very nice
+And from him I learned different commands
```
Comment (Hamma111) [inline task/t3.txt:2]: for it*

Comment (Hamma111) [reply task/t3.txt:2]: him is used for males. it is used for things

File: task/t4.txt [hunk 2]
```diff
@@ -0,0 +1,7 @@
+Sir I have searched nano its very good even better than vim
+And I learned various commands from him, the best thing was that there is no need to remember his commands, everything is shown in front.
+Switching from one file to another is easy۔
```
Comment (Hamma111) [inline task/t4.txt:3]: how do we save the file? you were supposed to explain that

**Review by Hamma111 — CHANGES_REQUESTED** [review]

**Review by Hamma111 — APPROVED** [review]

**Comment by asifnaseebi** [comment]
t3,t4[18march]

**Pull request merged**